### ISSUE TITLE
C3 runtime iOS export - black bar on iOS Devices

### ISSUE THREAD

**Issue by bubbblebub** [Scirra/Construct-3-bugs#2460]
## Problem description

When I export my project to iOS/ xcode, I have a black bar on the screen (screenshot is added in the zip file)

## Attach a .c3p
[iOS_export_Bug.zip](https://github.com/Scirra/Construct-3-bugs/files/2685579/iOS_export_Bug.zip)

## Steps to reproduce

1.export to iOS, all 3 export options enabled, min version 12+, x-code, hide status bar enabled
2.open in x code, preview on attached device (I tested on Iphone SE and IPad)
3.

## Observed result

pink screen with black bar on the screen boarder appears

## Expected result

no black bar

## First affected release

project made with r131

## Affected browsers/platforms

* Chrome: yes

## System details

<details><summary>View details</summary>

Platform information
Browser: Chrome
Browser version: 70.0.3538.110
Browser engine: Blink
Browser architecture: (unknown)
Context: browser
Operating system: Mac OS X
Operating system version: 10.14_1
Operating system architecture: (unknown)
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 8
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.110 Safari/537.36
C3 release: r131 (stable)
Language setting: en-US

Local storage
Storage quota (approx): 31 gb
Storage usage (approx): 955 mb (3%)
Persistant storage: No

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Supports highp precision: yes
Vendor: ATI Technologies Inc.
Renderer: AMD Radeon Pro 560 OpenGL Engine
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 2048
Extensions:

EXT_color_buffer_float
EXT_disjoint_timer_query_webgl2
EXT_texture_filter_anisotropic
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
Audio information
System sample rate: 44100 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

</details>

**Comment by AshleyScirra** [comment]
@shortercode - is this fixed now with Cordova 9?

**Comment by bubbblebub** [comment]
I just made an export of my project with the latest stable release R 148, unfortunately the black bar is still there

**Comment by AshleyScirra** [comment]
@bubbblebub - when you test things please only check the latest beta release, as we've already made a lot of changes since r148.

**Comment by shortercode** [comment]
I'm thinking this is probably the same as #2840, which has a patch incoming in the next release. So I would appreciate if you could check if there's any change.

**Comment by AshleyScirra** [comment]
r156 is now out, please test again!

**Comment by bubbblebub** [comment]
ok I made the test with R156. its much better but still there. When I started the bugreport Project from xcode on my Iphone, the black bar appeared from time to time. Then I disconnected my Iphone from the computer and then I had to restart maybe 20 times, but then there the black bar was again. I tested on an Iphone SE iOS 12.3.1. 
Xcode 10.2.1. Also on an IPad the black bar was there (but here I tested with my personal project, not with the Bugreport file). Activated in Xcode is "HIde status bar" and "Requires full screen" . The Bug appears with just one device orientation activated, or with both, so this seems not tho have any influence.

**Comment by AshleyScirra** [comment]
@bubbblebub - there's another change that may help with this in r156.2. Can you check again there to see if it is working?

**Comment by bubbblebub** [comment]
Its weird, I cant give you a clear answer: when I installed/ started the app with Xcode on the connected device the black bar is there quite often. But when I disconnect the device and start the app directly on the device, I couldnt reproduce it, no black bar. I tried quite often, start, closing the app completly, restart. I tested on Iphone SE and IPad. I will observe it a bit on my project in the future. Thanks for the effort!
